packages/run-protocol/src/vpool-xyk-amm/types.js

```diff
@@ -62,6 +62,8 @@
  * @property {(brand: Brand) => {toCentral: PriceAuthority, fromCentral: PriceAuthority}} getPriceAuthorities
  * get a pair of PriceAuthorities { toCentral, fromCentral } for requesting
  * Prices and notifications about changing prices.
+ * @property {() => Brand[]} getAllPoolBrands
+ * @property {() => Allocation} getProtocolPoolBalance
```

Comment:
I'm curious what prompted this. It looks like a bit of a (welcome) drive-by.

Reply:
I didn't investigate, but I suspected that moving tests around caused a change to the way declarations were imported or how eslint was configured in tests.  As I recall, we have eslint warnings turned off in some test directories.